plugins/csp-vr-accessibility/README.md

```diff
@@ -41,7 +39,7 @@ This plugin can be enabled with the following configuration in your `settings.js
 }
 ```
 
-Most of the configuration (all options, except for Grid's `falloff` and `texture`) is also available at runtime in the "VR Accessibility" tab in the settings menu.
+Most of the configuration (all options, except for Grid's `offset` and `texture`) is also available at runtime in the "VR Accessibility" tab in the settings menu.
```

Comment:
shouldn't is be extent instead of offset?
the offset is the transform node, and should be modifiable at runtime.
only the Texture and size of the Quad is only modifiable within the settings.json and therefore need to reload the plugin to be modified.